src/resolvers/mixEffect.ts

```diff
@@ -22,10 +22,8 @@ export function resolveMixEffectsState (oldState: StateObject, newState: StateOb
 		const newMixEffect = newState.video.ME[mixEffectId]
 		if (!oldMixEffect || !newMixEffect) continue
 		if (typeof newMixEffect.input !== 'undefined' && typeof newMixEffect.transition !== 'undefined') {
-			if (typeof oldMixEffect.input === 'undefined') {
-				oldMixEffect.input = oldMixEffect.programInput
-			}
-			if (newMixEffect.input !== oldMixEffect.input || newMixEffect.transition === Enums.TransitionStyle.DUMMY) {
+			const oldMEInput = oldMixEffect.input || oldMixEffect.programInput
```

Comment:
Should this be a `||`? Wont that 'reject' black (as that is 0)?

Reply:
Yes, if the old input is black (0) and the programInput is undefined it would always send the commands for the new input which is only a problem if the new input is also black and even then it doesn't hurt much. But good catch either way.

Reply:
If only there was a `??` operator in this language :smile: